src/digitalmarketplace/components/banner/template.njk

```diff
@@ -0,0 +1,17 @@
+<div
+    data-module="dm-banner"
```

Comment:
We're probably not going to need to use JavaScript with the banner component, so the data-module attribute is not needed

Reply:
Al said that it is better to keep it in.

Reply:
Its a way to namespace components and be consistent with components that do use JavaScript.

Reply:
Thanks mate :)

@lfdebrux Keep it? lose it?

Reply:
There's no harm in keeping it.